pkg/operator2/deployment.go

```diff
@@ -71,6 +71,14 @@ func defaultDeployment(
 			path:      ocpBrandingSecretMount,
 			keys:      []string{configv1.LoginTemplateKey, configv1.ProviderSelectionTemplateKey, configv1.ErrorsTemplateKey},
 		},
+		{
+			name:      trustedCABundleLocalName,
+			configmap: true,
+			path:      trustedCABundleMountDir,
+			mappedKeys: map[string]string{
```

Comment:
Why are we adding this complexity?  Keep `key` and `path` the same.

Reply:
Looking at the proxy doc, it seems to want this specific path for whatever reason - but I think the whole thing is pointless, see above.